src/test/functional/assertions.ts

```diff
@@ -1,4 +1,8 @@
 export const loginPageAssertion = {
     adminName: "Admin User",
     userName: "Normal User"
+}
+
```

Comment:
nie lepiej trzymać asercje w teście?

Reply:
dla mnie obojetnie jak chcesz to moge zmienic

Reply:
to zostawmy to na później, jak się bedziemy na siebie mergować to ustalimy